File: C#/BluffinMuffin.Protocol/Lobby/CheckCompatibilityResponse.cs
using System.Collections.Generic;
using BluffinMuffin.Protocol.DataTypes;
using BluffinMuffin.Protocol.DataTypes.Attributes;
using BluffinMuffin.Protocol.DataTypes.Enums;

namespace BluffinMuffin.Protocol.Lobby
{
    /// <summary>
    /// 
    /// </summary>
    public class CheckCompatibilityResponse : AbstractBluffinReponse<CheckCompatibilityCommand>
    {
        /// <summary>
        /// The version of the Implemented Bluffin Protocol by the client, Represented as "Major.Minor.Revision"
        /// </summary>
        [ExampleValue("2.0.0")]
        public string ImplementedProtocolVersion { get; set; }

        /// <summary>
        /// The LobbyTypes available on the server
        /// </summary>
        [ExampleValues(2, LobbyTypeEnum.QuickMode, LobbyTypeEnum.RegisteredMode)]
        public LobbyTypeEnum[] SupportedLobbyTypes { get; set; }

        /// <summary>
        /// The different games available on the server
        /// </summary>
        [ExampleValues(1)]
        public GameInfo[] AvailableGames { get; set; }

        /// <summary>
        /// 
        /// </summary>
        /// <param name="command"></param>
        public CheckCompatibilityResponse(CheckCompatibilityCommand command)
            : base(command)
        {
        }
    }
}

File: Python/BluffinMuffin.Protocol/lobby/check_compatibility_response.py
from abstract_response import AbstractResponse
from data_types.enums.lobby_type_enum import LobbyTypeEnum
from data_types.rule_info import RuleInfo
from lobby.check_compatibility_command import CheckCompatibilityCommand
from lobby.create_table_command import CreateTableCommand

__author__ = 'ericmas001@gmail.com'



class CheckCompatibilityResponse(AbstractResponse):
    def __init__(self, obj):
        super().__init__(obj, CheckCompatibilityCommand(obj['Command']))
        self.implemented_protocol_version = obj['ImplementedProtocolVersion']
        self.supported_lobby_types = [LobbyTypeEnum.parse(x) for x in obj['SupportedLobbyTypes']]
        self.rules = [RuleInfo(x) for x in obj['Rules']]

    def __str__( self ):
        return '{0} => {1} ({2}) ({3})'.format(super().__str__(), self.implemented_protocol_version, ', '.join([LobbyTypeEnum.to_string(x) for x in self.supported_lobby_types]), ', '.join([x.__str__() for x in self.rules]))